protocols/rendezvous/src/client.rs

```diff
@@ -281,6 +281,7 @@ impl NetworkBehaviour for Behaviour {
 
                     return Poll::Ready(new_to_swarm);
                 }
+                Poll::Ready(_) => return Poll::Pending,
```

Comment:
```suggestion
                Poll::Ready(_) => continue,
```

We should poll for more events instead.